cluster/defaults.go

```diff
@@ -86,6 +86,7 @@ type ExternalFlags struct {
 	ConfigDir        string
 	CustomCerts      bool
 	DisablePortCheck bool
+	DisableTimeCheck bool
```

Comment:
do you think we should enable it in types?

Reply:
This is identically setup as the port check, if it needs to be changed, both need to be moved.